src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Ноутбуки и компьютеры
+            </a>
```

Comment:
The best practice for this is:
1. Attributes should be on the new line and have 2 spaces
2. The closing bracket should be aligned as an open tag
3. Check it and fix in all task, please.
```suggestion
            <a
              href="#"
              class="navigation__link"
              data-qa="nav-hover"
            >
              Ноутбуки и компьютеры
            </a>
```